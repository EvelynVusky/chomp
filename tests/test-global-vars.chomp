string s = "hi";
int b = 4;

void foo() {
    println(b);
    println(s);    
    println(c);
}

int c = 5;

int main() {
    println(b);
    println(s);
    println(c);
    foo();
    return 0;
}