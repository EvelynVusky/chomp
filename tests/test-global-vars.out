4
hi
5
4
hi
5
